### PR TITLE
[FEATURE] Manage delayed event

### PR DESCRIPTION
**Pull request by pdenis**
Event process command
Event model
Default config
save event subscriber

### CODE REVIEW

File: MhpsAsyncBundle.php [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Itkg\DelayEventBundle;
+
+use Symfony\Component\HttpKernel\Bundle\Bundle;
+
+class MhpsDelayEventBundle extends Bundle
```
Comment (juchi) [inline MhpsAsyncBundle.php]: incorrect naming

**Comment by pdenis** [comment]
ping @juchi

**Comment by juchi** [comment]
:+1:

File: EventDispatcher/DelayEventDispatcher.php [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+        $this->dispatcher = $dispatcher;
+        $this->eligibleEventNames = $eligibleEventNames;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function dispatch($eventName, Event $event = null)
+    {
+        echo $eventName;
```
Comment (ghost) [inline EventDispatcher/DelayEventDispatcher.php]: to be deleted ?

**Comment by ghost** [comment]
:+1:

**Pull request merged**